### PR TITLE
SC-19936 update windows services

### PR DESCRIPTION
**Pull request by cpipilas**
Update Windows Monitoring integration:
- Added section for Windows Services
- Misc updates throughout the page

### CODE REVIEW

File: docs/images/integrations/windows-top-services.png
Comment (boratanrikulu) [inline docs/images/integrations/windows-top-services.png:1]: This screenshot looks a weird. Somehow charts seem to have multiple line?

Comment (cpipilas) [reply docs/images/integrations/windows-top-services.png:1]: Can you elaborate Bora? It seems 1-1 to me :)

Comment (boratanrikulu) [reply docs/images/integrations/windows-top-services.png:1]: This one for example 
image

Comment (cpipilas) [reply docs/images/integrations/windows-top-services.png:1]: Aha, because it's grouped by host.

**Pull request merged**